.github/workflows/cli-core-audit.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [16.x, 14.x, 12.x, 10.x]
```

Comment:
Any reason to drop 10/12, beyond them no longer being LTS? Do they no longer work?

Reply:
No, LTS was the major reason. 12x would still work as-is, acc to the compatibility [matrix](https://oclif.io/blog/2022/01/12/announcing-oclif-v2#compatibility-matrix) for oclif v2. 
10x would need to be tested.